collectors/logsmanagement.plugin/plugin_logsmanagement.c

```diff
@@ -0,0 +1,411 @@
+
+        struct File_info *p_file_info = p_file_infos_arr->data[i];
+        if(!p_file_info->parser_config){ // Check if there is parser configuration to be used for chart generation
+            chart_data_arr[i] = NULL;
+            continue; 
+        } 
+
+        /* Circular buffer memory stats - add dimensions */
+        stats_chart_data->dim_circ_buff_mem_total_arr[i] = 
+            rrddim_add( stats_chart_data->st_circ_buff_mem_total, 
+                        p_file_info->chart_name, NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+        stats_chart_data->dim_circ_buff_mem_uncompressed_arr[i] = 
+            rrddim_add( stats_chart_data->st_circ_buff_mem_uncompressed, 
+                        p_file_info->chart_name, NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+        stats_chart_data->dim_circ_buff_mem_compressed_arr[i] = 
+            rrddim_add( stats_chart_data->st_circ_buff_mem_compressed, 
+                        p_file_info->chart_name, NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+
+        /* Compression stats - add dimensions */
+        stats_chart_data->dim_compression_ratio[i] = 
+            rrddim_add( stats_chart_data->st_compression_ratio, 
+                        p_file_info->chart_name, NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+
+        /* DB disk usage stats - add dimensions */
+        stats_chart_data->dim_disk_usage[i] = 
+            rrddim_add( stats_chart_data->st_disk_usage, 
+                        p_file_info->chart_name, NULL, 1, 1, RRD_ALGORITHM_ABSOLUTE);
+
+        chart_data_arr[i] = callocz(1, sizeof(struct Chart_meta));
+        memcpy(chart_data_arr[i], &chart_types[p_file_info->log_type], sizeof(struct Chart_meta));
+        chart_data_arr[i]->init(p_file_info, chart_data_arr[i]);
+
+        /* Custom charts - initialise */
+        for(int cus_off = 0; p_file_info->parser_cus_config[cus_off]; cus_off++){
+            chart_data_arr[i]->chart_data_cus_arr = reallocz( chart_data_arr[i]->chart_data_cus_arr, 
+                                                              (cus_off + 1) * sizeof(Chart_data_cus_t *));
+            chart_data_arr[i]->chart_data_cus_arr[cus_off] = callocz(1, sizeof(Chart_data_cus_t));
+
+            RRDSET *st_cus = rrdset_find_active_bytype_localhost( p_file_info->chart_name, 
+                                                                  p_file_info->parser_cus_config[cus_off]->chart_name);
+            if(st_cus) chart_data_arr[i]->chart_data_cus_arr[cus_off]->st_cus = st_cus;
+            else {
+                chart_data_arr[i]->chart_data_cus_arr[cus_off]->st_cus = rrdset_create_localhost(
+                        (char *) p_file_info->chart_name
+                        , p_file_info->parser_cus_config[cus_off]->chart_name
+                        , NULL
+                        , "regex charts"
+                        , NULL
+                        , p_file_info->parser_cus_config[cus_off]->chart_name
+                        , "matches/s"
```

Comment:
Please, remove `/s` from the metric.

Reply:
@thiagoftsm I've been thinking about this and I am not sure about the change. While what you say is correct, I can see other collectors using `/s` , so I added in the first place for consistency.